### PR TITLE
Update CI checks

### PR DESCRIPTION
**Pull request by Bolodya1997**
1. `sdk-vppagent` should be replaced with `sdk-vpp` in dependencies check.
2. `go generate` check should check not only `*.gen.go` files.

Refers to #61.

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -85,40 +85,39 @@ jobs:
...
-        run: go get -u github.com/golang/protobuf/protoc-gen-go@v1.3.3
-      - name: Install proto-gen-go
-        run: go get github.com/searKing/golang/tools/cmd/go-syncmap
-      - name: Generate files
-        run: go generate ./...
-      - name: Check for changes in generated code
+      - name: Instal go generate tools
+        run: |
+          go get -u github.com/golang/protobuf/protoc-gen-go@v1.3.3
+          go get github.com/searKing/golang/tools/cmd/go-syncmap
```
Comment (denis-tingaikin) [inline .github/workflows/ci.yaml:113]: We are not using these in cmd.

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -85,40 +85,39 @@ jobs:
...
+        env:
+          GO111MODULE: on
+      - uses: actions/checkout@v2
```
Comment (denis-tingaikin) [inline .github/workflows/ci.yaml:116]:
```suggestion
```

File: .github/workflows/ci.yaml [hunk 3]
```diff
@@ -85,40 +85,39 @@ jobs:
...
         run: |
-          git diff --name-only --exit-code go.mod || ( echo "Run go tidy" && false )
-          git diff --name-only --exit-code go.sum || ( echo "Run go tidy" && false )
+          git diff --name-only --exit-code || ( echo "Run go mod tidy" && false )
 
   gogenerate:
     name: Check generated files
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
```
Comment (denis-tingaikin) [inline .github/workflows/ci.yaml:105]: Why?

File: .github/workflows/ci.yaml [hunk 4]
```diff
@@ -85,40 +85,39 @@ jobs:
...
       - uses: arduino/setup-protoc@master
         with:
           version: '3.8.0'
       - uses: actions/setup-go@v1
         with:
           go-version: 1.15
```
Comment (denis-tingaikin) [inline .github/workflows/ci.yaml:109]:
```suggestion
```

**Review by denis-tingaikin — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (Bolodya1997) [reply .github/workflows/ci.yaml]: OK, removed :)

**Review by denis-tingaikin — APPROVED** [review]

**Pull request merged**